vector/src/main/java/im/vector/adapters/VectorMessagesAdapter.java

```diff
@@ -1430,43 +1430,30 @@ private View getImageVideoView(int type, final int position, View convertView, V
             Event event = row.getEvent();
             Message message = null;
 
-            int waterMarkResourceId = -1;
-
+            boolean videoContent = false;
             if (type == ROW_TYPE_IMAGE) {
-
                 ImageMessage imageMessage = JsonUtils.toImageMessage(event.getContent());
-
-                if ("image/gif".equals(imageMessage.getMimeType())) {
-                    waterMarkResourceId = R.drawable.filetype_gif;
+                if (imageMessage.getMimeType().equals("image/gif")) {
+                    videoContent = true;
                 }
                 message = imageMessage;
-
             } else if (type == ROW_TYPE_VIDEO) {
-
+                videoContent = true;
                 message = JsonUtils.toVideoMessage(event.getContent());
-                waterMarkResourceId = R.drawable.filetype_video;
-
             } else if (type == ROW_TYPE_STICKER) {
-
                 StickerMessage stickerMessage = JsonUtils.toStickerMessage(event.getContent());
                 message = stickerMessage;
             }
 
-            // display a type watermark
-            final ImageView imageTypeView = convertView.findViewById(R.id.messagesAdapter_image_type);
-
-            if (null == imageTypeView) {
+            // display a play icon for video content
+            final ImageView playCircleView = convertView.findViewById(R.id.messagesAdapter_play_circle);
+            playCircleView.setVisibility(View.GONE);
```

Comment:
this will crash if playCircleView is null. Please to this after the next if (as it was done before)

Reply:
Done.